requirements.txt

```diff
@@ -4,4 +4,5 @@ colorzero==2.0
 gpiozero==1.6.2
 protobuf==3.15.6
 dbus-python==1.2.16
+nmcli==0.5.0
```

Comment:
@ryanteck what version of `nmcli` is the above code from?

Reply:
Diff vs upstream nmcli

(.env) robe8437@Roberts-MacBook-Pro nmcli % diff -r nmcli nmcli_nebra     
diff -r nmcli/_system.py nmcli_nebra/_system.py
38c38
<                           check=True, env={'LANG': 'C'})
---
>                           check=True)
Only in nmcli_nebra/data: __pycache__
(.env) robe8437@Roberts-MacBook-Pro nmcli %